e2e/tests/content-history/history.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { test, expect } from '@playwright/test';
+import { login } from '../../utils/login';
+import { resetDatabaseAndImportDataFromPath } from '../../scripts/dts-import';
+import { describeOnCondition } from '../../utils/shared';
+
+const hasFutureFlag = process.env.STRAPI_FEATURES_FUTURE_CONTENT_HISTORY === 'true';
+
+describeOnCondition(hasFutureFlag)('History', () => {
+  test.describe('Collection Type', () => {
+    test.beforeEach(async ({ page }) => {
+      await resetDatabaseAndImportDataFromPath('./e2e/data/with-admin.tar');
+      await page.goto('/admin');
+      await login({ page });
+    });
+
+    test('A user should be able create, edit, or publish/unpublish an entry, navigate to the history page, and select versions to view from a list', async ({
+      page,
+    }) => {
+      // Navigate to the content-manager - collection type - article
+      await page.getByRole('link', { name: 'Content Manager' }).click();
+      await page.getByRole('link', { name: /Create new entry/, exact: true }).click();
+      await page.waitForURL(
+        '**/content-manager/collection-types/api::article.article/create?plugins\\[i18n\\]\\[locale\\]=en'
+      );
+
+      /**
+       * Create
+       */
+      const titleInput = await page.getByRole('textbox', { name: 'title' });
+      await titleInput.fill('Being from Kansas is a pity');
+      await page.getByRole('button', { name: 'Save' }).click();
+      // Go to the history page
+      await page.getByRole('link', { name: 'History' }).click();
+      await page.waitForURL(
+        '**/content-manager/collection-types/api::article.article/*/history?plugins\\[i18n\\]\\[locale\\]=en'
```

Comment:
Could you also test that you're auto redirected to `&id=versionId`? then you can delete [this file](https://github.com/strapi/strapi/blob/7463352479ed8ceab21a1a3ad6e16d1a528ad012/packages/core/admin/admin/src/content-manager/history/pages/tests/History.test.tsx#L2)

Reply:
Yes I'll take a look. That also reminds me I will update the waitForUrls to use a regex Josh sent me